cidemipyme-gatsby/src/components/sections/MetodologiesSection.jsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Container,
+  Col,
+  Row,
+} from 'reactstrap';
+import HoverCard from 'components/HoverCard.jsx';
+
+import metodology_cce from 'images/metodologia_cce.png';
+import metodology_business_architecture from 'images/metodologia_arquitectura_negocio.png';
+import metodology_growth_strategy from 'images/metodologia_estrategia_crecimiento.png';
+import metodology_cidetalento from 'images/metodologia_cidetalento.png';
+
+const metodologiesKeyPrefix = 'metodologiesSection_';
+const metodologiesImages = {
+  'cce': metodology_cce,
+  'business_architecture': metodology_business_architecture,
+  'growth_strategy': metodology_growth_strategy,
+  'cidetalento': metodology_cidetalento,
+};
+
+
+const MetodologyItemPropType = {
+  id: PropTypes.string.isRequired,
+  title: PropTypes.string.isRequired,
+  content: PropTypes.arrayOf(PropTypes.string).isRequired
+}
+
+const MetodologiesPropTypes = {
+  title: PropTypes.string.isRequired,
+  metodologies: PropTypes.arrayOf(PropTypes.shape(MetodologyItemPropType)).isRequired,
+}
+
+class MetodologiesSection extends React.Component {
+  renderContent(content, keyPrefix) {
+    return content.map((text, index) => {
+      return (
+        <div key={`${keyPrefix}${index}`}>
+          <p>{text}</p>
+        </div>
+      );
+    });
+  }
+
+  render() {
+    let cards = [];
+    let totalCards = this.props.data.metodologies.length;
+    let key = `${metodologiesKeyPrefix}card_`;
+
+    for(let i=0; i < totalCards; i+=2) {
+      cards.push(
+        <Row className="row-padding-top-30">
+          <Col md={6} key={`${key}${i}`}>
+            <HoverCard
+              title={this.props.data.metodologies[i].title}
+              text={this.props.data.metodologies[i].content}
+              image={metodologiesImages[this.props.data.metodologies[i].id]}
+              keyPrefix={`${key}${i}`}/>
```

Comment:
why don't you use the id as the key prefix (`this.props.data.metodologies[i].id`)?

Reply:
just to keep it constant for that class. In case I want to reuse it....